src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Static methods is a bad practice. Seems like it fits there, but if in the future ColorSupplier could have some options for creation or dynamic fields, maybe it is better to make method non-static?